prescription/prescription.go

```diff
@@ -0,0 +1,127 @@
+package prescription
+
+import (
+	"github.com/tidepool-org/platform/data/types/settings/pump"
+)
+
+const (
+	PrescriptionStateDraft     = "draft"
+	PrescriptionStatePending   = "pending"
+	PrescriptionStateSubmitted = "submitted"
+	PrescriptionStateReviewed  = "reviewed"
+	PrescriptionStateExpired   = "expired"
+	PrescriptionStateActive    = "active"
+	PrescriptionStateInactive  = "inactive"
+
+	PrescriptionTrainingInPerson = "inPerson"
+	PrescriptionTrainingInModule = "inModule"
+
+	PrescriptionTherapySettingInitial              = "initial"
+	PrescriptionTherapySettingTransferPumpSettings = "transferPumpSettings"
+	PrescriptionTherapySettingCPT                  = "cpt" // TODO: Certified Personal Trainer?
+
+	PrescriptionLoopModeClosedLoop  = "closedLoop"
+	PrescriptionLoopModeSuspendOnly = "suspendOnly"
+)
+
+
+type Prescription struct {
```

Comment:
I think we need to support digitally signing the prescriptions - at a minimum with the prescriber's key, but ideally with the patient's key as well. Neither have keys at present time.